jmclient/jmclient/blockchaininterface.py

```diff
@@ -390,7 +388,53 @@ def estimate_fee_per_kb(self, N):
             return 10000
         return int(Decimal(1e8) * Decimal(estimate))
 
-class BitcoinCoreNoHistoryInterface(BitcoinCoreInterface):
+class RegtestBitcoinCoreMixin():
```

Comment:
It's a litte complicated of course, but I suspect this is a good solution to the problem of handling the fact that we now have different variants of Core interfaces that can be regtest or not.